docs/guides/Integration-Guides/login-integration-guide/registering-credentials-to-set-up-apple-id-login.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Registering credentials to set up Apple ID login"
+slug: "registering-credentials-to-set-up-apple-id-login"
+hidden: false
+createdAt: "2023-09-05T17:34:17Z"
+updatedAt: "2023-09-05T17:34:17Z"
+---
+
+> This feature is in closed beta, which means that only specific customers can access it now. If you want to implement it in the future, please contact [our support](https://support.vtex.com/hc/pt-br/).
+
+To allow users to [sign in to your store with their Apple ID](https://developer.apple.com/sign-in-with-apple/get-started/), you must create valid Apple credentials before [setting up this type of authentication through the VTEX Admin](https://help.vtex.com/en/tutorial/configuring-sign-in-with-apple-id-beta--5qprgEmHYfPTghnYwm0KrV). This guide describes all the necessary steps to get these credentials:
+
+- [Before you start](#before-you-start)
+- [Instructions](#instructions)
+  - [Step 1: Creating an App ID](#step-1-creating-an-app-id)
+  - [Step 2: Creating a Services ID](#step-2-creating-a-services-id)
+  - [Step 3: Creating a private Key ID and p8 Certificate](#step-3-creating-a-private-key-id-and-p8-certificate)
+  - [Step 4: Configuring the Apple ID login in the VTEX Admin](#step-4-configuring-the-apple-id-login-in-the-vtex-admin)
+
+After following these steps to register Apple credentials, you must complete the configuration of Apple ID login in the VTEX Admin, as explained in [Configuring Sign in with Apple ID](https://help.vtex.com/en/tutorial/configuring-sign-in-with-apple-id-beta--5qprgEmHYfPTghnYwm0KrV).
+
+> Read [Apple's documentation](https://developer.apple.com/help/account/configure-app-capabilities/create-a-sign-in-with-apple-private-key) for more information about the required credentials.
+
+## Before you start
+
+You must have an [Apple ID](https://support.apple.com/apple-id) and be a member of the [Apple Developer Program](https://developer.apple.com/programs/) to follow the instructions presented in this guide.
+
+## Instructions
+
+### Step 1: Creating an App ID
+
+Get started by enabling the Sign in with Apple service on an iOS, tvOS, watchOS, or macOS App ID and classifying it as the primary App ID. Read [Apple’s documentation](https://developer.apple.com/help/account/configure-app-capabilities/about-sign-in-with-apple) on this step for more information.
```

Comment:
Esse poderia talvez ser o passo 1 desse step? Ou já que a pessoa precisa do sign in, talvez seja uma passo no before you start. Daí nesse step 1 a pessoa já começa com "1. Sign in to the..."

Reply:
Sugestão: deixar esse texto como um before you start e no lugar adicionar uma intro para o step 1. Ex.: "Begin by creating the App ID that will serve as a..."

Reply:
```suggestion
Begin by following the instructions below to enable the Sign in with Apple service on an iOS, tvOS, watchOS, or macOS App ID and classifying it as the primary App ID. Read [Apple’s documentation](https://developer.apple.com/help/account/configure-app-capabilities/about-sign-in-with-apple) on this step for more information.
```